src/test/java/com/epam/tc/hw1/TestCalcSub.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.tc.hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class TestCalcSub {
+
+    @Test(groups = "sub and sum", dataProvider = "subLongCorrectData", dataProviderClass = DataProviderTest.class)
+    public void subTwoDigits(long firstDigit, long secondDigit, long result) {
+        Calculator calculator = new Calculator();
```

Comment:
`new Calculator();` It's a code duplication. Minimize it using before, after annotation